Two-body-scf-guess.py

```diff
@@ -0,0 +1,126 @@
+from wrap_pyscf import *
+import numpy as np
+
+def make_monomer_dens(monomers, bs):
+    monomer_dens = []
+    for j in monomers:
+        k = density(j, bs)
+        monomer_dens.append(k)
+    return monomer_dens
+
+def make_list_of_ddls(dimers, bs):
+    list_of_ddls = []
+    for d_lists in dimers:
+        list_of_ddls.append([])
+        for dimer in d_lists:
+            dens = density(dimer, bs)
+            list_of_ddls[-1].append(dens)
+    return list_of_ddls
+  
+def make_z_rho(m, a):
+    z_rho = np.zeros((m*a, m*a))
+    return z_rho
+
+def make_ei_list(list_of_ddls):
+    ei_list = []
+    for dd_lists in list_of_ddls:
+        ei_list.append([])
+    for s in ei_list:
+        s.append([])
+        s.append([])
+    return ei_list
+
+def fill_ei_list(list_of_ddls, ei_list, m):
+    i_list = ei_list
+    i_list[0][0].append(np.zeros((m*2, m*2)))
+    i_list[-1][1].append(np.zeros((m*2, m*2)))
+    for s in i_list:
+        s[0].append(np.zeros((m*2, m*2)))
+        s[1].append(np.zeros((m*2, m*2)))
+    for i, dd_lists in enumerate(list_of_ddls):
+        for j, dens in enumerate(dd_lists):
+            if j < i:
+                i_list[i][0].append(dens)
+            if j >= i:
+                i_list[i][1].append(dens)
+    return i_list
+
+def add_interacts(i_list, m):
+    overlap = []
+    for s in i_list:
+        s0 = sum(s[0])[m:m*2, m:m*2]
+        s1 = sum(s[1])[0:m, 0:m]
+        ol = np.add(s0, s1)
+        overlap.append(ol)
+    return overlap
+
+def compute_diags(overlap, monomer_dens, a):
+    diags = []
+    for i, ol in enumerate(overlap):
+        diag = overlap[i] - ((a-2)*monomer_dens[i])
+        diags.append(diag)
+    return diags
+
+def rho_diagonal(diags, z_rho, m):
+    for i, dgs in enumerate(diags):
+        z_rho[i*m:(i+1)*m, i*m:(i+1)*m] = diags[i]
+    rho_d = z_rho
+    return rho_d
+
+def rho(list_of_ddls, rho_d, m):
+    for i, dd_lists in enumerate(list_of_ddls):
+        for j, dens in enumerate(dd_lists):
+            while j >= i:
+                rho_d[i*m:(i+1)*m, (j+1)*m:(j+2)*m] = list_of_ddls[i][j][0:m, m:m*2]
+                rho_d[(j+1)*m:(j+2)*m, i*m:(i+1)*m] = list_of_ddls[i][j][m:2*m, 0:m]
+                break
+    rho = rho_d
+    return rho
+rho = rho(list_of_ddls, rho_d, m)
```

Comment:
You missed moving this line out of the function section. Also do not use a variable with the same name as a function.